lti_tool/templates/lti_tool/lti_submission.html

```diff
@@ -10,11 +10,8 @@
 {% block siblings_bottom %}{% endblock %}
 {% block footer %}{% endblock %}
 
-{% block exercise_wait %}
-{% include "lti_tool/_lti_exercise_wait.html" %}
-{% endblock %}
-
 {% block exerciseinfo %}
 {{ block.super }}
 {% include 'exercise/_submission_info.html' %}
```

Comment:
```suggestion
```
Since this template inherits `submission.html`, `{{ block.super }}` already includes `submission_info.html`. Including it twice causes the same box to appear twice in the rendered html page.